Dockerfile

```diff
@@ -14,15 +14,25 @@ Run dnf config-manager --set-enabled powertools \
     && dnf install -y @container-tools \
     && dnf install -y jq
 
-# install python 3.9 - take several minutes
-RUN dnf install -y python3.9 \
-    && echo alias python=python3.9 >> ~/.bashrc
+# Prerequisite for Python installation
+ARG python_version=3.10
+ARG python_full_version=3.10.8
+RUN dnf install openssl-devel bzip2-devel wget -y
+
+# Install Python
+RUN wget https://www.python.org/ftp/python/${python_full_version}/Python-${python_full_version}.tgz \
+    && tar -xzf Python-${python_full_version}.tgz \
```

Comment:
Again, you don't need a temporary file here.

Reply:
See above reply